### PR TITLE
#669 js test compilation

### LINKED ISSUE #669: Fix CI checks for 0.24.0 release
Some of CI checks failed on ubuntu with JS tests. we should fix it before release

### PR DESCRIPTION
**Pull request by ExNDY**
fix #669

### CODE REVIEW

File: resources-generator/src/main/kotlin/dev/icerock/gradle/MultiplatformResourcesPlugin.kt [hunk 1]
```diff
@@ -89,6 +92,20 @@ open class MultiplatformResourcesPlugin : Plugin<Project> {
...
+                    compilation.binaries.withType<JsIrBinary>()
+                        .configureEach { binary: JsIrBinary ->
+                            binary.linkTask.configure { linkTask ->
+                                setupJsExecutableResources(
+                                    linkTask = linkTask,
+                                    projectDir = project.provider { project.projectDir }
+                                )
+                            }
+                        }
+                }
```
Comment (Alex009) [inline resources-generator/src/main/kotlin/dev/icerock/gradle/MultiplatformResourcesPlugin.kt:106]: lets move it to function, like already done with 
```
if (target is KotlinNativeTarget) {
                setupExecutableResources(target = target)
                setupFrameworkResources(target = target)
                 setupTestsResources(target = target)
             }
```

**Review by Alex009 — CHANGES_REQUESTED** [review]

**Review by Alex009 — APPROVED** [review]

File: samples/resources-gallery/web-app/build.gradle.kts [hunk 2]
```diff
@@ -17,6 +17,7 @@ kotlin {
 
                 implementation(compose.html.core)
                 implementation(compose.runtime)
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.8.1-Beta")
```
Comment (Alex009) [inline samples/resources-gallery/web-app/build.gradle.kts:20]: why this should be added?

Comment (ExNDY) [reply samples/resources-gallery/web-app/build.gradle.kts:20]: After upgrade sample with: kotlin 1.9.23 and compose multiplatform 1.6.1 for this block:
`
val color by remember(window) {
                    Testing.getGradientColors().first().getColorFlow(window)
                }.collectAsState(initial = MokoColor(0xFFAAFFFF))
`
I'm get error: idea can't identify collectAsState, after add coroutines i'm get success build and run

Comment (ExNDY) [reply samples/resources-gallery/web-app/build.gradle.kts:20]: I think need set stable version

File: samples/android-mpp-app/app/build.gradle.kts [hunk 3]
```diff
@@ -34,7 +34,6 @@ android {
 
 kotlin {
     androidTarget()
-    ios()
```
Comment (Alex009) [inline samples/android-mpp-app/app/build.gradle.kts:37]: why it was removed?

File: samples/android-mpp-app/gradle.properties [hunk 4]
```diff
@@ -10,5 +10,3 @@ kotlin.mpp.androidSourceSetLayoutVersion=2
 kotlin.mpp.androidGradlePluginCompatibility.nowarn=true
 
 android.useAndroidX=true
-
-xcodeproj=./sample/ios-app
```
Comment (Alex009) [inline samples/android-mpp-app/gradle.properties:14]: why?

**Review by Alex009 — CHANGES_REQUESTED** [review]

**Pull request merged**